action.yaml

```diff
@@ -2,11 +2,16 @@ name: Upload Charm Docs
 description: Upload charm documentation to charmhub
 author: David Andersson
 inputs:
-  create_new_topic:
+  delete_topics:
     description: |
-      Whether to create a new topic and commit it back to the repository if the
-      docs key in metadata.yaml either does not exist or is empty. Defaults to
-      not creating a new topic in which case an error is returned.
+      Whether to delete topics from the documentation server if they are no
+      longer needed.
+    default: true
+    required: false
+    type: boolean
+  draft_mode:
```

Comment:
I suggest we rename this to `dry-run`. It seems clearer to me.

Reply:
Good suggestion, fixed